manager/controllers/baseline_update.go

```diff
@@ -70,6 +70,7 @@ func BaselineUpdateHandler(c *gin.Context) {
 		return
 	}
 
+	inventoryIDsList := map2list(req.InventoryIDs)
 	missingIDs, err := checkInventoryIDs(account, map2list(req.InventoryIDs))
```

Comment:
replace `map2list(req.InventoryIDs)`  with `inventoryIDsList` so it does not run map2list() twice